### PR TITLE
#7 create a basic page with hardcoded data of a list of countries and display those values using static generation

### PR DESCRIPTION
**Pull request by jeandreidc**
Resolve issue #7 
- Added new page for travel packages that can be accessed through the route `/travel-packages` 
- Added external method that a page uses to pre-load assets or data

Screenshot: 
image

### CODE REVIEW

File: src/app/lib/travel-packages.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            itinerary: [
+                {
+                    dayNo: 0,
+                    activities: ['swimming', 'bungee jumping']
+                }
+            ]
+        }
+    ];
+}
+
```
Comment (samchuu) [inline src/app/lib/travel-packages.ts:76]: Test

**Review by samchuu — CHANGES_REQUESTED** [review]
Minor changes

File: src/app/lib/travel-packages.ts (hunk 1 above)
Comment (jeandreidc) [reply src/app/lib/travel-packages.ts:76]: I think this should be fine

**Review by samchuu — APPROVED** [review]
Looks good to me!

**Pull request merged**